src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -1,51 +1,73 @@
 package com.volunteer.enterprise;
-import dto.Volunteer;
+import com.volunteer.enterprise.dto.Volunteer;
+import com.volunteer.enterprise.service.IVolunteerService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 
+import java.util.List;
+
 @Controller
 public class VolunteerAppController {
+    IVolunteerService volunteerService;
 
- @Autowired
- VolunteerService volunteerService;
-
+    /**
+     *RequestMapping for root (/) endpoint
+     * Create a Voluntier volunteer object then display start
+     * @return Voluntier index page
+     */
 
     @RequestMapping("/")
     public String index(Model model){
 
-Volunteer volunteer = new Volunteer();
-
-volunteer.setVolunteerID(1);
-volunteer.setVolunteerUser("Iron Man");
-volunteer.setVolunteerPass("1234");
+    Volunteer volunteer = new Volunteer();
+    volunteer.setVolunteerID(1);
+    volunteer.setVolunteerUser("Iron Man");
+    volunteer.setVolunteerPass("1234");
+    model.addAttribute(volunteer);
 
-model.addAttribute(volunteer);
-        return "Index";
+        // Add newly created favorite to model so it can be displayed
+        model.addAttribute(volunteer);
+        return "start";
```

Comment:
In the controller, I had to return start.html instead of Index.html to make code work. I would recommend fixing Index.html or combining it with the start.html.

Reply:
Why was that change needed?  This method is returning a string, so what is in the string should not matter.  The first thing I'd check: Capitalization.  The first approach returned Index, which means the page should be called Index.html, not index.html.  This is not common.